featuretools/primitives/standard/aggregation_primitives.py

```diff
@@ -14,7 +14,7 @@
 )
 
 from featuretools.primitives.base.aggregation_primitive_base import AggregationPrimitive
-from featuretools.utils import convert_time_units
+from featuretools.utils import calculate_trend, convert_time_units
```

Comment:
How does this work? Aren't these utils in time_utils.py?

Reply:
I took out the guts of `Trend` in agg prims, slightly modified it so it could be used for the Rolling version, and placed it into `calculate_trend`. I took `calculate_trend` its helper functions away from `aggregation_primitive.py` into `time_utils.py` since I felt like that's where they kind of belonged based on the name of the file.